apps/buildproof_dapp/app/utils/submit-hackathon/validation.ts

```diff
@@ -0,0 +1,39 @@
+import type { Prize } from './types';
+
+export const isFormValid = (
+  partnerName: string,
+  hackathonTitle: string,
+  description: string,
+  startDate: string,
+  endDate: string,
+  totalCashPrize: number,
+  totalPrizeAmount: number
+): boolean => {
+  const isAllFieldsFilled =
+    partnerName !== '' &&
+    hackathonTitle !== '' &&
+    description !== '' &&
+    startDate !== '' &&
+    endDate !== '' &&
+    totalCashPrize > 0;
+
+  const isTotalCorrect = totalPrizeAmount === totalCashPrize;
+
+  return isAllFieldsFilled && isTotalCorrect;
+};
+
+export const validateDates = (startDate: string, endDate: string): boolean => {
```

Comment:
this return a boolean, convention is to name the function starting with the verbe, eg : 
```ts 
areDatesValid ( )
```